docs/Maths.md

```diff
@@ -47,6 +47,76 @@ Result:
 
 ![](matplotlib.png)
 
+### Using Pandas ###
+Create a plot using [pandas.DataFrame.plot](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html)
+
+```python
+from fpdf import FPDF
+import pandas as pd
+import matplotlib.pyplot as plt
+import io
+
+data = {'Unemployment_Rate': [6.1,5.8,5.7,5.7,5.8,5.6,5.5,5.3,5.2,5.2],
+        'Stock_Index_Price': [1500,1520,1525,1523,1515,1540,1545,1560,1555,1565]
+       }
+
+plt.figure() # Store figure
+df = pd.DataFrame(data,columns=['Unemployment_Rate','Stock_Index_Price'])
+df.plot(x ='Unemployment_Rate', y='Stock_Index_Price', kind = 'scatter')
+
+# Converting Figure to an image:
+img_buf = io.BytesIO() # Create image object
+plt.savefig(img_buf, dpi=200) # Save the image
+
+pdf = FPDF()
+pdf.add_page()
+pdf.image(img_buf, w=pdf.epw)
```

Comment:
Following some users feedback, I realized that `.epw` is a bit of a jargon acronym,
and that its meaning is not really not obvious.
Hence, could you please add a comment on this line like `# full width` ?
This applies to any other usage of `.epw` on this documentation page 🙂

Reply:
No problem, I will add this comment to other `.epw`